src/selectors/layout.js

```diff
@@ -38,26 +39,34 @@ export const getSidebarWidth = (visibleSidebar, outerChartWidth) => {
   return sidebarWidth.closed;
 };
 
+/**
+ * Return the displayed width of the meta sidebar
+ */
+export const getMetaSidebarWidth = visibleMetaSidebar =>
+  visibleMetaSidebar ? metaSidebarWidth.open : metaSidebarWidth.closed;
```

Comment:
This might help with the mobile screen width issues that I mentioned previously:
```suggestion
export const getMetaSidebarWidth = (visibleMetaSidebar, outerChartWidth) => {
  if (visibleMetaSidebar && outerChartWidth > sidebarBreakpoint) {
    return metaSidebarWidth.open;
  }
  return metaSidebarWidth.closed;
};
```